diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
index 4c506533866d411b18799e5441f02c2b3e9a0c44..87f8fbf2ed793510eaa2450364515319c0d8920a 100755
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -239,6 +239,8 @@ build_slice() {
     SWIFT_COMPILATION_MODE=wholemodule \
     CLANG_ENABLE_CODE_COVERAGE=NO \
     CLANG_COVERAGE_MAPPING=NO \
+    CODE_SIGNING_ALLOWED=NO \
+    CODE_SIGNING_REQUIRED=NO \
   )
 
   local product_path="${BUILD_PRODUCTS_PATH}/${build_dir_name}"
